src/giftCards/GiftCardsList/GiftCardsListOrderCard/GiftCardsListOrderCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { Link, makeStyles, Typography } from "@material-ui/core";
+import { Alert } from "@saleor/macaw-ui";
+import { orderGiftCardBoughtPath } from "@saleor/orders/urls";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { giftCardListOrderCardMessages as messages } from "./messages";
+
+const useStyles = makeStyles(
```

Comment:
We could just remove these styles and add a vertical spacer to the parent component